### DISCUSSION TITLE
Brats MRI Segmenation Model

### DISCUSSION THREAD

**Discussion by SameerShanbhogue** [Project-MONAI/model-zoo#289] [Q&A]
Hi 

Why brats_mri_segmentation is not listed in modelbundle models

Invalid Model(s) are provided: ['brats_mri_segmentation'], either not in model zoo or not supported with MONAI Label
Following are the available models.  You can pass comma (,) separated names to pass multiple
Available bundle with latest tags:
   -c models
        lung_nodule_ct_detection 
        pancreas_ct_dints_segmentation 
        prostate_mri_anatomy 
        renalStructures_UNEST_segmentation 
        spleen_ct_segmentation 
        spleen_deepedit_annotation 
        swin_unetr_btcv_segmentation 
        wholeBrainSeg_Large_UNEST_segmentation
Or provide valid local bundle directories

**Comment by yiheng-wang-nv** [comment]
Hi @SameerShanbhogue , I checked the MONAI Label source code, and the `brats_mri_segmentation` is not in the maintained list (https://github.com/Project-MONAI/MONAILabel/blob/269213b7760c0aff1e75f6b593a47968555d2423/monailabel/utils/others/modelzoo_list.py#L25).
Hi @tangy5 , do you know the reason why it is not maintained? Thanks in advance.
